css-transforms-2/Overview.bs

```diff
@@ -1106,6 +1106,65 @@ The transform functions <<matrix()>>, ''matrix3d()'' and ''perspective()'' get c
 
 For interpolations with the primitive ''rotate3d()'', the direction vectors of the transform functions get normalized first. If the normalized vectors are equal, the rotation angle gets interpolated numerically. Otherwise the transform functions get converted into 4x4 matrices first and interpolated as defined in section <a href="#matrix-interpolation">Interpolation of Matrices</a> afterwards.
 
+Addition and accumulation of transform lists {#combining-transform-lists}
+============================================
+
+<a lt="value addition">Addition</a> of transform lists is
+defined as <a spec=infra>list</a> concatenation
+such that <var ignore>V<sub>result</sub></var> is equal to
+<var ignore>V<sub>b</sub></var> <a spec=infra for=list>appended</a> to
+<var ignore>V<sub>a</sub></var>.
+
+<a lt="value accumulation">Accumulation</a> of transform lists
+follows the same steps as interpolation
+with regards to matching transform functions including
+padding lists with <a>identity transform functions</a>,
+converting ''transform/none'' to an <a>identity transform function</a>,
+and converting both arguments to matrices as necessary (see
+[[css-transforms-1#interpolation-of-transforms]]).
+However, instead of interpolating the individual parameters,
+they are combined using arithmetic addition--
+except in the case of parameters whose value is one in the
+<a>identity transform function</a>
+(e.g. scale parameters and matrix elements <var ignore>m11</var>, <var
+ignore>m22</var>, <var ignore>m33</var>, and <var ignore>m44</var>),
+which combine using <dfn>accumulation for one-based values</dfn> as follows:
+
+  <var ignore>V<sub>result</sub></var> =
+  <var ignore>V<sub>a</sub></var> + <var ignore>V<sub>b</sub></var> - 1
```

Comment:
These two lines are separated and indented in the source, but all this will do in the output is make them inline in a new paragraph. Was that intended?

Also, again with using `<div algorithm>`, and defining the input vars to your algorithm and then using them; that way you won't need to `ignore` them all.

(and note that you can use `|V|` instead of `<var>V</var>`, to make the var usage less intrusive.)

Reply:
This too is referencing the notation from css-values-4. I guess we should work out how to link it somehow?

Unfortunately `|V<sub>result</sub>|` doesn't parse as a variable so we have to use `<var>V<sub>result</sub></var>`.

Reply:
Ah, yes, markup will indeed disturb that. All the *other* variables can be done simpler. ^_^

Reply:
Oh, I missed the other variables. I tried switching `m11` etc. to `|m11|` but then I got warnings about unreferenced variables. I found another reference to `<var ignore>m11</var>` in the doc and dropped the `ignore` attribute but it didn't fix it because I had scoped my `m11` to the "transform list accumulation" algorithm.

I wonder if there's some way to make the "Mathematical Description of Transform Functions" section define these variables? They're currently defined in an image there.